PYME/LMVis/Extras/surface_fitting.py

```diff
@@ -44,7 +47,6 @@ def OnFitQuadraticPatches(self, event):
         
         self.vis_frame.Refresh()
 
-
-def Plug(visFr):
+def Plug(vis_frame):
```

Comment:
```suggestion
def Plug(visFr):
```

The current `visFr` variable name is gross, but an existing convention. `vis_frame`  as an alternative is only marginally less gross and breaks convention. If and when we change this it should be done globally, with appropriate changes to the plugin docs, and to a name which is actually sane.